src/models/Authority.ts

```diff
@@ -0,0 +1,22 @@
+import BaseModel from '@/models/_BaseModel'
+
+export default class Authority extends BaseModel {
+  public declare id: string
```

Comment:
probably not important, but only authority has integer for id
```suggestion
  public declare id: number
```